src/main/kotlin/no/nav/tilleggsstonader/sak/interntVedtak/InterntVedtak.kt

```diff
@@ -65,3 +92,27 @@ data class Stønadsperiode(
     val fom: LocalDate,
     val tom: LocalDate,
 )
+
+data class VilkårInternt(
```

Comment:
Kan navnet `Internt` bli forvirrende? Kanskje bedre å bruke `VilkårInterntVedtak` eller `VilkårBlankett` f.eks. hvis den første blir for lang. Så ingen plutselig får opp denne som forslag og tenker "uu for internt bruk" 🤠

Reply:
Sannsynligvis tilstrekkelig forvirrende for å skjønne at man ikke burde bruke det :-)

Reply:
Føler ikke at noen av de 2 alternativen er noe bedre dessverre :/ Blankett burde vi iaf ikke bruke